recipes/patchelf/all/conanfile.py

```diff
@@ -1,5 +1,10 @@
-from conans import ConanFile, AutoToolsBuildEnvironment, tools
-from conans.errors import ConanInvalidConfiguration
+from conan import ConanFile
+from conan.tools.apple import is_apple_os
+from conan.tools.files import get, chdir, rmdir
+from conan.tools.scm import Version
+from conans import AutoToolsBuildEnvironment
+from conan.errors import ConanInvalidConfiguration
+import conans
```

Comment:
```suggestion
from conans import tools as legacy_tools
```
We will migrate other tools in the future. Legacy_tools helps us to find the technical debt with a grep command